hack/lib/build/constants.sh

```diff
@@ -2,8 +2,16 @@
 
 # This script provides constants for the Golang binary build process
 
-readonly OS_BUILD_ENV_GOLANG="${OS_BUILD_ENV_GOLANG:-1.7}"
-readonly OS_BUILD_ENV_IMAGE="${OS_BUILD_ENV_IMAGE:-openshift/origin-release:golang-${OS_BUILD_ENV_GOLANG}}"
+OS_BUILD_ENV_GOLANG="${OS_BUILD_ENV_GOLANG:-1.7}"
+OS_BUILD_ENV_IMAGE="${OS_BUILD_ENV_IMAGE:-openshift/origin-release:golang-${OS_BUILD_ENV_GOLANG}}"
+
+if [[ "$(go env GOARCH)" == "ppc64le" ]]; then
```

Comment:
I would love to but we have some issues with 1.7.x release and fixes went into 1.8. Even in k8s we are generating the binaries using golang 1.8.